keyboards/redox_w/redox_w.h

```diff
@@ -8,15 +8,14 @@
 #define blu_led_on    PORTF &= ~(1<<4)
 #define grn_led_off   PORTD |= (1<<1)
 #define grn_led_on    PORTD &= ~(1<<1)
+#define wht_led_off   PORTD |= (1<<0)
+#define wht_led_on    PORTD &= ~(1<<0)
 
-#define set_led_off     red_led_off; grn_led_off; blu_led_off
-#define set_led_red     red_led_on;  grn_led_off; blu_led_off
-#define set_led_blue    red_led_off; grn_led_off; blu_led_on
-#define set_led_green   red_led_off; grn_led_on;  blu_led_off
-#define set_led_yellow  red_led_on;  grn_led_on;  blu_led_off
-#define set_led_magenta red_led_on;  grn_led_off; blu_led_on
-#define set_led_cyan    red_led_off; grn_led_on;  blu_led_on
-#define set_led_white   red_led_on;  grn_led_on;  blu_led_on
```

Comment:
why are these removed?

Reply:
As you can see, the keyboard contains only 4 LEDs (default D1: red, D2: blue, D3: grene, D4: white). 
Each led has its own switch, not RGB style, which means that white led on does not mean red and blue and green are on at the same time. 
So I think these define is useless.
![led_pic](https://github.com/mattdibi/redox-keyboard/blob/master/img/redox-w-receiver-assembly-3.jpg)